tests/marine/test_marine.py

```diff
@@ -1025,3 +1025,7 @@ def test_parse_fields_preserves_order(marine_instance: Marine, tcp_packet: bytes
         "udp.srcport": None,
         "tcp.srcport": "16424",
     }
+
+
+def test_filter_packet_larger_than_1514(marine_instance: Marine):
+    assert not marine_instance.filter(b"a" * 2500, display_filter="frame contains test")
```

Comment:
I think you should use a valid packet here to make sure dissection works properly for big packets